### ISSUE TITLE
[Feature Suggestion] force to show effects from latest update

### ISSUE THREAD

**Issue by 0rangeGhost** [gta-chaos-mod/ChaosModV#1180]
would be cool if new effects, that is added in an update would be forced to activate at least once in first 30-60 minutes of playing the game. because of nature of random - it is possible that you can never experience new effect, so you can miss some cool stuff

**Comment by anto4440** [comment]
Yeah, you could make like a 'new only' option, so that you could test the new effects
